### PR TITLE
Market feed item

### PR DESCRIPTION
**Pull request by styliii**
market index card layout

TODO: there's a bug that the graph doesn't immediately draw.  You have to tap NDX or SPX.  will fix later.

### CODE REVIEW

File: WealthNOW/.idea/gradle.xml [hunk 1]
```diff
@@ -5,7 +5,6 @@
       <GradleProjectSettings>
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="1.7" />
```
Comment (mikeng13) [inline WealthNOW/.idea/gradle.xml]: This is probably not good.

File: WealthNOW/app/src/main/java/com/finnerds/wealthnow/helpers/ChartHelper.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import android.graphics.Color;
+
+import com.github.mikephil.charting.charts.LineChart;
+import com.github.mikephil.charting.components.XAxis;
+import com.github.mikephil.charting.components.YAxis;
+
+/**
+ * Created by Li on 4/12/15.
+ */
+public class ChartHelper {
```
Comment (mikeng13) [inline WealthNOW/app/src/main/java/com/finnerds/wealthnow/helpers/ChartHelper.java]: Nice!

**Comment by mikeng13** [comment]
Looks good. Just one small thing.

File: WealthNOW/.idea/gradle.xml (hunk 1 above)
Comment (styliii) [inline WealthNOW/.idea/gradle.xml]: ugh!  this is so annoying that these things keep slipping in, even though it's in gitignore.  any ideas?

Comment (mikeng13) [inline WealthNOW/.idea/gradle.xml]: Nope we gotta eventually figure it out lol

**Comment by styliii** [comment]
redid the layout @mikeng13 and fixed gradle file

**Pull request merged**